packages/components/src/components/icon/icons.js

```diff
@@ -175,6 +175,7 @@ import './common/ic-bank-dashboard.svg';
 import './common/ic-bank.svg';
 import './common/ic-bell.svg';
 import './common/ic-black-warning.svg';
+import './common/ic-blocked-user.svg';
```

Comment:
This can just be ic-block, since it's not related to a user or P2P for that matter.

Reply:
okiess